Sources/EdgeNetworkHandlers/NetworkResponseHandler.swift

```diff
@@ -20,13 +20,18 @@ import Foundation
 class NetworkResponseHandler {
     private let LOG_TAG = "NetworkResponseHandler"
     private let serialQueue = DispatchQueue(label: "com.adobe.edge.eventsDictionary") // serial queue for atomic operations
+    private let dataStore = NamedCollectionDataStore(name: EdgeConstants.EXTENSION_NAME)
```

Comment:
What do you think about moving the last reset over to StoreResponsePayloadManager as it already has access to the DataStore and this change is related to the store cleanup?

Reply:
We could do that, but given our timeline, I'd prefer not to as there is very little benefit.